components/auth/BankIdAuth.tsx

```
@@ -143,6 +129,9 @@ export function BankIdAuth({ mode, onComplete, hero = false }: BankIdAuthProps)
   const lastStartRef = useRef<number>(0)
   const onCompleteRef = useRef(onComplete)
   onCompleteRef.current = onComplete
+  /** Latest status, readable from long-lived event listeners. */
+  const statusRef = useRef(status)
+  statusRef.current = status
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/components/auth/BankIdAuth.tsx:134:3
  132 |   /** Latest status, readable from long-lived event listeners. */
  133 |   const statusRef = useRef(status)
> 134 |   statusRef.current = status
      |   ^^^^^^^^^^^^^^^^^ Cannot update ref during render
  135 |
  136 |   const pollFailureCount = useRef(0)
  137 |   /** True while a poll response is being processed — prevents overlapping ticks. */

(react-hooks/refs)